source/index.html.md.erb

```diff
@@ -388,6 +388,63 @@ This core hierarchy of Sheet > Column > Row > Cell is essential to working with
 
 # Sample Code and SDKs
 
+> Install the SDK
+
+```javascript
+npm install smartsheet
+```
+
+```csharp
+Install-Package smartsheet-csharp-sdk
+```
+
+```java
+git clone https://github.com/smartsheet-platform/smartsheet-java-sdk.git
```

Comment:
This compiles from source - so not symmetrical with the others. Should be the add dependency fragment

Reply:
Done